extensions/adobe/experience/consumer-experienceevent.schema.json

```diff
@@ -9,7 +9,7 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Consumer ExperienceEvent  Mixin",
   "type": "object",
-  "description": "This mixin groups a set of standard fields to describe the behavior of an individual. This mixin can be used to express behavior of a consumer related to digital content consumption (web, mobile app), online or off-line purchases. The use of this standard represention allows for a single representation for data producers and consumers of consumer behavior in Experience Platform",
+  "description": "This field group is a set of standard fields to describe the behavior of an individual. This field group can be used to express behavior of a consumer related to digital content consumption (web, mobile app), online or off-line purchases. The use of this standard represention allows for a single representation for data producers and consumers of consumer behavior in Experience Platform",
```

Comment:
Looks like the title of this needs to be changed...to consumer experienceEvent Field Group

Description could be shortened to " A set of ...